metrics/web/test/common/presentation/metrics_theme/model/metrics_table/theme_data/project_metrics_table_theme_data_test.dart

```diff
@@ -31,6 +32,17 @@ void main() {
       },
     );
 
+    test(
```

Comment:
Maybe we can place these checks for default parameters into one test as we do for the Light and Dark theme? What do you think?